### PR TITLE
Merged libesedb, libevt, libevtx, libexe, liblnk, libmsiecf and libregf into single project

### PR DESCRIPTION
**Pull request by joachimmetz**
Context for reviewer: https://github.com/google/oss-fuzz/pull/4160

### CODE REVIEW

File: projects/libyal/build.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+for PROJECT in ${SRC}/*;
```
Comment (inferno-chromium) [inline projects/libyal/build.sh:18]: Most of the lines don't need to end with semicolon, can remove those.

File: projects/libyal/build.sh [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+do
+	PROJECT=$(basename ${PROJECT});
```
Comment (inferno-chromium) [inline projects/libyal/build.sh:20]: 2 spaces instead of 8. Same other lines.

File: projects/libyal/build.sh [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+	make -j$(nproc) LIB_FUZZING_ENGINE=${LIB_FUZZING_ENGINE}
+
+	# Download the test data if supported.
+	if test -x ./synctestdata.sh;
+	then
+		./synctestdata.sh;
+	fi
+
+	# Copy the fuzzer binaries and test data to the output directory.
+	for FUZZER_TARGET in $(cd ossfuzz && find . -executable -type f);
```
Comment (inferno-chromium) [inline projects/libyal/build.sh:44]: usually we call this FUZZ_TARGET instead of FUZZER_TARGET.

File: projects/libyal/build.sh (hunk 2 above)
Comment (joachimmetz) [reply projects/libyal/build.sh:20]: done, tabs replaced by 2 spaces

File: projects/libyal/build.sh (hunk 3 above)
Comment (joachimmetz) [reply projects/libyal/build.sh:44]: done

File: projects/libyal/build.sh (hunk 1 above)
Comment (joachimmetz) [reply projects/libyal/build.sh:18]: ack, removed several semicolons

**Pull request merged**